src/pages/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from "react";
 
 export default function Home() {
   return (
-    <button className="text-4xl px-8 py-2 text-skin-primary">
+    <button className="text-4xl px-8 py-2 text-skin-primary w-full md:h-full">
```

Comment:
As with the border above, h-screen doesn't mesh with window resizing _within_ a browser on a PC. Does `w-full` calculate the browsers width?